### PR TITLE
feat(vim-alias): add full screen help

### PR DESCRIPTION
**Pull request by georgettica**
I don't use it very often and am fine with not putting it in, but worth pushing it out there

### CODE REVIEW

**Comment by NoahGorny** [comment]
hey @georgettica
Seems like you had trailing whitespaces in your code, fix it up and I will happily merge this.

PS: Would you like to clean this alias file as part of #1696?  
This would be great :smile:

**Comment by georgettica** [comment]
I can do that, but not sure how to run the pre-commit check

**Comment by NoahGorny** [comment]
You need to add your file to `clean_files.txt` and run `lint_cleam_files.sh`

We will add docs about it asap so it will be clearer

**Comment by georgettica** [comment]
I was actually looking at the pt you referenced me in.
Will run it

**Comment by georgettica** [comment]
```
$ lint_clean_files.sh
./lint_clean_files.sh: line 23: pre-commit: command not found
```

**Comment by NoahGorny** [comment]
@georgettica seems like you need to install `pre-commit`. Usually this can be achieved by running `python -m pip install pre-commit`

Also make sure to install `shfmt`, which is the formatter we use in Bash-it

**Comment by georgettica** [comment]
Okidok.
Thanks for pointing it out :))

**Comment by georgettica** [comment]
@NoahGorny added what you requested now :) hope it's good enough

**Comment by NoahGorny** [comment]
@georgettica amazing work! Thanks for your contribution :heart:

File: aliases/available/vim.aliases.bash [hunk 1]
```diff
@@ -5,15 +5,19 @@ VIM=$(command -v vim)
 GVIM=$(command -v gvim)
 MVIM=$(command -v mvim)
 
-[[ -n $VIM ]] && alias v=$VIM
+# open the vim help in fullscreen incorporated from
+# https://stackoverflow.com/a/4687513
+alias vimh='${VIM} -c ":h | only"'
```
Comment (NoahGorny) [inline aliases/available/vim.aliases.bash:10]: I think you should put this inside a `if [[ -n $VIM]]` like what's being done with the `v` alias

**Review by NoahGorny — COMMENTED** [review]
Left a small comment, otherwise looks great

File: aliases/available/vim.aliases.bash (hunk 1 above)
Comment (georgettica) [reply aliases/available/vim.aliases.bash:10]: Oh, fair comment. Will do and hopefully as you Invision it

File: aliases/available/vim.aliases.bash [hunk 2]
```diff
@@ -5,15 +5,18 @@ VIM=$(command -v vim)
 GVIM=$(command -v gvim)
 MVIM=$(command -v mvim)
 
-[[ -n $VIM ]] && alias v=$VIM
+[[ -n $VIM ]] && alias v='$VIM'
+# open the vim help in fullscreen incorporated from
+# https://stackoverflow.com/a/4687513
+[[ -n $VIM ]] && alias vimh='${VIM} -c ":h | only"'
```
Comment (NoahGorny) [inline aliases/available/vim.aliases.bash:11]: why not do something like this:
```bash
if [[ -n $VIM ]]; then
    alias v='$VIM'
    alias vimh=  '${VIM} -c ":h | only"'
fi

**Review by NoahGorny — APPROVED** [review]
Great work @georgettica !

File: aliases/available/vim.aliases.bash (hunk 2 above)
Comment (nwinkler) [reply aliases/available/vim.aliases.bash]: Agree - this is much cleaner and easier to read as well.

**Comment by georgettica** [comment]
Yay!

**Pull request merged**